.github/workflows/ci_tests.yaml

```
@@ -62,14 +62,19 @@ jobs:
             isDraft: true
           - os: windows-latest
             isDraft: true
-        # Pair Python 3.10 with NumPy 1.23 and Python 3.12 with NumPy 1.26
-        # Only install optional packages on Python 3.12/NumPy 1.26
+        # Pair Python 3.10 with the minimum supported versions of NumPy/Pandas/Xarray
+        # and Python 3.12 with the latest versions of NumPy/Pandas/Xarray
+        # Only install optional packages on Python 3.12
         include:
           - python-version: '3.10'
             numpy-version: '1.23'
+            pandas-version: '1.5'
+            xarray-version: '2022.03'
             optional-packages: ''
           - python-version: '3.12'
             numpy-version: '1.26'
+            pandas-version: '2.2'
+            xarray-version: '2024.01'
```

Following what we are doing with NumPy 1.26, we should also pin pandas and xarray to their latest versions here. But, it means we have to watch the releases of these core package dependencies and update these versions very often (they usually have one minor release every few months).

So, instead of tracking the latest releases, we can just pin Python to the latest version, and conda/mamba should automatically install the latest numpy/pandas/xarray versions that are compatible with the pinned Python version. Does it sounds OK?

---

Ok with not testing the newest versions for `pandas` and `xarray`, but we'll still need to set a minimum pin for the older version I think.

```suggestion
            pandas-version: ''
            xarray-version: ''
```